### PR TITLE
General design issues

### PR DESCRIPTION
**Pull request by MostafaBalata**

### CODE REVIEW

File: page/sass/common.scss [hunk 1]
```diff
@@ -55,11 +55,12 @@ body{
         padding: 0 0 $space-bottom-between-elements $space-bottom-between-elements;
         li {
             color: black;
-            font-weight: $font-weight-bold;
+            font-weight: $font-weight-bold;
```
Comment (krzkaczor) [inline page/sass/common.scss]: what kind of editor you use? you're leaving a lot of trash like this in source code

Comment (krzkaczor) [inline page/sass/common.scss]: plase dont commit things like this

File: page/ts/index.ts [hunk 2]
```diff
@@ -60,5 +60,17 @@ $(document).ready(function(){
     $('.team .see-more').click(function(){        
         $(this).text().trim().toLowerCase() == seeMore.trim().toLowerCase() ? $(this).text(seeLess) : $(this).text(seeMore);
         $('.team .is-hidden').fadeToggle("slow","linear");        
-    });
+    });    
+});
+
+$(window).scroll(function(e){
```
Comment (krzkaczor) [inline page/ts/index.ts]: great! could you also change color of that border to #f0f0f1? There is a variable called `$navbar-default-border`

File: page/sass/ecosystem.scss [hunk 3]
```diff
@@ -24,19 +24,24 @@
         top: 10px;
         color: $blue-darker;
         margin-bottom: 3px;
+        margin-left: 36px;
         .underline {
             margin-top: 1px;
             border-bottom: 5px solid $blue-darker;
             width: 36px;
         }
+        h5 {
```
Comment (krzkaczor) [inline page/sass/ecosystem.scss]: was there any ticket to change font weight?

Comment (MostafaBalata) [inline page/sass/ecosystem.scss]: It Is in the design

Comment (MostafaBalata) [inline page/sass/ecosystem.scss]: Technology, Social Impact and Regulatory Efforts the are lights

**Pull request merged**